resources/assets/build/webpack.config.js

```diff
@@ -92,6 +92,7 @@ let webpackConfig = {
             { loader: 'resolve-url', options: { sourceMap: config.enabled.sourceMaps } },
             {
               loader: 'sass', options: {
+                implementation: require('sass'),
                 sourceMap: config.enabled.sourceMaps,
                 sourceComments: true,
```

Comment:
I think we can drop this line since `sourceComments` aren't going to be supported in the JavaScript API for Dart Sass.